br/pkg/streamhelper/advancer.go

```diff
@@ -435,7 +435,7 @@ func (c *CheckpointAdvancer) onTaskEvent(ctx context.Context, e TaskEvent) error
 		}
 		log.Info("get global checkpoint", zap.Uint64("checkpoint", globalCheckpointTs))
 		c.lastCheckpoint = newCheckpointWithTS(globalCheckpointTs)
-		p, err := c.env.BlockGCUntil(ctx, globalCheckpointTs)
+		p, err := c.env.BlockGCUntil(ctx, globalCheckpointTs-1)
```

Comment:
can you comment why minus the gc safepoint by 1? is there any bug if we specify `globalCheckpointTs` instead of `globalCheckpointTs-1`?

Reply:
This will be fixed in the following up pr

Reply:
Noticed that otherwhere we use `BlockUntil`, the safepoint is set to `c.lastCheckpoint.safeTS()`.
That is:
```
func (c *checkpoint) safeTS() uint64 {
	return c.TS - 1
}
```

Therefore, we need to keep it sync. Otherwise, when we resume a task and the `GC safepoint == globalCheckpointTs`, the tick will try to block `c.TS -1` and fail.